server/controllers/CookieController.js

```diff
@@ -2,19 +2,44 @@ const db = require('../models/Models');
 
 const CookieController = {};
 
-// getUserByEmail
-
```

Comment:
I was not sure if this was an attempt at creating a new middleware function or documentation for setSSIDCookie. 

Regardless, I think my comments suffice for documentation.